helm/charts/determined/templates/genai/genai-shared-fs-permissions-script.yaml

```diff
@@ -0,0 +1,22 @@
+{{- if .Values.genai }}
+{{- if .Values.genai.version }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: genai-shared-fs-permissions-script-{{ .Release.Name }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: genai-{{ .Release.Name }}
+    release: {{ .Release.Name }}
+data:
+  enable_shared_fs.sh: |
+    #!/bin/sh -ex
+    echo "whoami: $(whoami)"
+    chmod 2775 /shared_fs
+    GROUP_ID={{ (required "A valid .Values.genai.generatedPVC.agentGroupID entry required!" .Values.genai.generatedPVC.agentGroupID) }}
+    GROUP_NAME={{ (required "A valid .Values.genai.generatedPVC.agentGroupName entry required!" .Values.genai.generatedPVC.agentGroupName) }}
+    groupadd -g $GROUP_ID $GROUP_NAME
```

Comment:
I am not sure about this line: I think admins are supposed to be adding posix groups through other means for it to work properly. let's ask: https://hpe-internal.slack.com/archives/C04R79C223B/p1711581093797589

Reply:
okay, based on the answer I'd suggest we remove `groupadd` here.

Reply:
Gotcha. There might be another way to do that. I only added the group specifically because chgrp wouldn't take a number on the chgrp available on the container. Maybe there's another one that's more updated or has that ability